modules/common/src/main/java/com/ibm/cloud/cloudant/common/SdkCommon.java

```diff
@@ -16,13 +16,13 @@
 import java.io.IOException;
```

Comment:
Can you update the copyright header to `© Copyright IBM Corporation 2020, 2022. All Rights Reserved.`?

Reply:
Done